### ISSUE TITLE
Experiment 3

### ISSUE THREAD

**Issue by jds8** [jds8/SAE4DLM-CE#7]
For experiment 3, we want a plot similar to experiment 1 but where SAE reconstructions are only used during a portion of the diffusion process. We expect to see that cross entropy delta is largest for the early stages of diffusion. For this experiment, we can fix a sparsity level (choose the sparsest level, 50, as in Figure 3) and generate a plot of cross-entropy delta vs diffusion stage (early, middle, and late). Generate data and plots for L1, L10, and L23.

**Comment by Dabsoysauce** [comment]
The experiment reveals that the importance and function of SAE features vary drastically depending on both the network depth (layer) and the diffusion stage. Early layer (L1) features appear to be consistently less critical, and removing them even improves the loss, particularly in late diffusion stages. In contrast, middle layer (L10) features are highly crucial during the late diffusion stage, as their removal causes a sharp increase in loss. Late layer (L23) features have a comparatively minimal impact on the loss when masked across all stages. This suggests a specialization where middle layers handle the critical representations needed for the final stages of the diffusion process.

<img width="790" height="490" alt="Image" src="https://github.com/user-attachments/assets/84c8c62a-9522-4d68-92e7-912d1f793991" />
